framework/log/FileTarget.php

```diff
@@ -131,8 +131,9 @@ public function export()
         }
         $writeResult = @fwrite($fp, $text);
         if ($writeResult === false) {
-            $error = error_get_last();
-            throw new LogRuntimeException("Unable to export log through file ({$this->logFile})!: {$error['message']}");
+            $message = "Unable to export log through file ($this->logFile)!";
```

Comment:
@samdark is there any reason why Yii::t was not used in error messages? 
Should it be left that way or addressed?

Reply:
`Yii::t()` is used only for messages and exceptions that should be displayed to end user (for example HTTP exceptions). I don't thing that these messages should be translated.

Reply:
@xcopy never mind. @rob006 yeah, I was wondering but your comment makes sense.